Evaluation/Input/Content/Section2.3_Model_Parameters_and_Assumptions.md

```diff
@@ -129,11 +129,10 @@ This is the result of the final parameter identification:
 | `Solubility at reference pH`                                | 2800 FIXED (see [Section 2.2.1](#221-In-vitro-and-physicochemical-data)) | mg/L      |
 | `kcat` AADAC (with a reference concentration of 1 µmol/L)   | 9.865                                                        | 1/min     |
 | `kcat` P-gp (with a reference concentration of 1.41 µmol/L) | 0.6088                                                       | 1/min     |
-| `kcat` OATP1B1 (with a reference concentration of 1 µmol/L) | 7.796                                                        | 1/min     |
+| `kcat` OATP1B1 (with a reference concentration of 1 µmol/L) | 7.796<sup>*</sup>                                            | 1/min     |
```

Comment:
the comment below says "The value was set to 5.210" but the value did not change in the table?